Taskfile.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: "3"
```

Comment:
Single quotes would be preferred.

Reply:
I revert Taskfile.yml from main branch.

I will add my 5 cents about YAML. I am assuming you meant something else when you told me about the preferred format. But anyway, when I need a string literal in YAML, I often use double quotes because most of the time I have to use escaping. I configured IDE formatter for this reason.

But in fact, when do we use single quotes? The answer is to avoid. There are few advantages, mostly inconvenience. You can read about [single quotes in YAML](https://yaml.org/spec/1.2-old/spec.html#style/flow/single-quoted). I found a link for you as TRDL. [How to write strings in YAML with different ways](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210). I agree with this author. In any case, it's up to you.

Reply:
Maybe this is relevant when the file contains data (lines with '). But if you look at the Taskfile you might notice double quotes in -ldflags. This means they would have to escape (It will also confuse). 
There are no wrong and right quotes. Almost always you can write like this:
```yaml
'go build -v -tags=UI -ldflags="-s -w" -o pb-dropbox-downloader.app .'
```
or like this:
```yaml
"go build -v -tags=UI -ldflags='-s -w' -o pb-dropbox-downloader.app ."
```
It is more logical to choose what is more convenient now and maintain consistency.